exopy_hqc_legacy/instruments/drivers/visa/tektro_awg.py

```diff
@@ -506,6 +507,36 @@ def to_send(self, name, waveform):
         self._driver.write_binary_values(header, waveform, datatype='B')
         self.write('*WAI')
 
+    @secure_communication()
+    def send_load_awg_file(self, awg_file, filename='setup', timeout=100):
```

Comment:
Since `awg_file` is actually byte data, I would prefer awg_file_content. Otherwise I would naively think of a path.